templates/documentation.html

```diff
@@ -0,0 +1,297 @@
+                GitHub
+              </button>
+              <button type="button" class="btn btn-default dropdown-toggle" data-toggle="dropdown" aria-haspopup="true"
+                      aria-expanded="false">
+                <span class="caret"></span>
+              </button>
+              <ul class="dropdown-menu dropdown-menu-normal">
+                <li class="dropdown-header">Other</li>
+                <li>
+                  <a href="#" class="btn-search" data-provider="google">
+                    <i class="fa fa-fw fa-flip-horizontal fa-search documentation-icon-dark"></i>
+                    Google
+                  </a>
+                </li>
+              </ul>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+
+  <section class="documentation-section section-content-white-bg">
+    <div class="container">
+      <div class="row">
+        <div class="col-lg-6">
+          <div class="row centered">
+            <span class="section-title">Getting help</span>
+            <div class="col-lg-12">
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-keyboard-o"></i> <a href="https://gitter.im/inveniosoftware/invenio">Chat</a>
+                  </h2>
+                  <p>
+                    Chat with Invenio developers
+                  </p>
+                </div>
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-comment"></i> <a href="https://invenio-talk.web.cern.ch/">Forum</a></h2>
+                  <p>
+                    Ask questions on the forum
+                  </p>
+                </div>
+              </div>
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2><i class="fa fa-users"></i> <a href="{{ '/people'|url }}">People</a></h2>
+                  <p>
+                    Review the list of contributors
+                  </p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-6">
+          <div class="row centered">
+            <span class="section-title">Contributing</span>
+            <div class="col-lg-12">
+              <div class="row">
+                <div class="col-sm-6">
+                  <h2>
+                    <i class="fa fa-compass"></i>
+                    <a href="https://invenio.readthedocs.io/en/latest/community/contribution-guide.html">
+                      Contribution guide
+                    </a>
+                  </h2>
+                  <p>
+                    Learn how to contribute to the project
+                  </p>
+                </div>
+                <div class="col-sm-6">
+                  <h2>
+                    <i class="fa fa-users"></i>
+                    <a href="https://invenio.readthedocs.io/en/latest/community/governance.html">Governance</a>
```

Comment:
We have this page inside this site already, see https://inveniosoftware.org/governance/. I think this is the one we should link.